### ISSUE TITLE
Enter-BCContainer on PowerShell 7 doesn't seem to work

### ISSUE THREAD

**Issue by pzentner** [microsoft/navcontainerhelper#3087]
PLEASE DO NOT INCLUDE ANY PASSWORDS OR TOKENS IN YOUR ISSUE!!!

**Describe the issue**
Running 

```
Enter-BcContainer -containerName [name]
```

results in an error message when run on PowerShell 7 (full output shown below)

**Scripts used to create container and cause the issue**
```
Enter-BcContainer -containerName [name]
```

**Full output of scripts**
```
PowerShell 7.3.4
PS C:\Users\PeterZentner> Enter-BcContainer -containerName [name]
BcContainerHelper version 5.0.4-preview1032
BC.HelperFunctions emits usage statistics telemetry to Microsoft
Running on PowerShell 7
New-PSSession: C:\Users\PeterZentner\Documents\PowerShell\Modules\BcContainerHelper\5.0.4\ContainerHandling\Get-NavContainerSession.ps1:60
Line |
  60 |  …  $session = New-PSSession -ContainerId $containerId -RunAsAdministrat …
     |                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     | The input ContainerId 23704982e45b7fcae5e961e07b93786c4936f4d6101bf9c1ccbf6c7f1ea34e99 does not exist, or the
     | corresponding container is not running.
```

**Screenshots**

**Additional context**
Happens all the time on PowerShell 7, but works without a problem on "old" Windows PowerShell

**Comment by f4n0** [comment]
Hi!
I've installed the prerelease version, but it seems not working!

 'Get-WmiObject' is not recognized

BcContainerHelper version 5.0.4-preview1037
BC.HelperFunctions emits usage statistics telemetry to Microsoft
Running on PowerShell 7
Get-BcContainerSession: C:\Users\sprimo\Documents\PowerShell\Modules\BcContainerHelper\5.0.4\ContainerHandling\Enter-NavContainer.ps1:22
Line |
  22 |  …  $session = Get-BcContainerSession -containerName $containerName -sil …
     |                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     | The term 'Get-WmiObject' is not recognized as a name of a cmdlet, function, script file, or executable program.
     | Check the spelling of the name, or if a path was included, verify that the path is correct and try again.


@freddydk

**Comment by freddydk** [comment]
Strange, somehow this worked on my machine - but it shouldn't.

**Comment by freddydk** [comment]
A fix is in the next prerelease - should be out in 1-2 hours

**Comment by f4n0** [comment]
Confirm!
it is working right!

Thanks

**Comment by freddydk** [comment]
Shipped in 5.0.4
